### ISSUE TITLE
Postfix: smtp capabilities 

### ISSUE THREAD

**Issue by chrisch-hh** [Mailu/Mailu#704]
I had the issue that sending mails from Windows Live Mail client was not possible and ended with an `530 5.7.1 client was not authenticated` error in the client. Other (more modern) mail clients that I've tested work without any problems (e.g. Thunderbird 60, Outlook 2016, Android Mail).

After some testing with `telnet mail.myserver.com 25` and `telnet mail.myserver.com 587` I realized, that after the EHLO command there is only one answer `250-mail.myserver.com`. The other 250 answers about the smtp server's capabilities are missing. Obviously the smtp server capabilities are not passed through the front nginx container to the client. But Window Live Mail seems to need the `250-AUTH PLAIN LOGIN` to establish an authenticated connection with the smtp server.

So I've added in the front containers nginx.conf `smtp_capabilities "SIZE 50000000" PIPELINING ENHANCEDSTATUSCODES 8BITMIME DSN VRFY ETRN "AUTH PLAIN LOGIN";` in the server's section for port 25 and 587. Telnet now shows the capabilities after EHLO and sending mails with Windows Live Mail is working. 

Is it the intended behavior of Mailu that the smtp server's capabilities are not passed through? Or have I missed something in the config? Any concerns about my solution? I'm not so deep into Mailu and SMTP protocol.

**Comment by developer011** [comment]
I ran into the same issue with Outlook for Mac Version 16.18 (181014).
Adding smtp_auth login to nginx.conf seems to fix the issue for Outlook for Mac.

Original:
`    server {
      listen 465 ssl;
      listen [::]:465 ssl;
      protocol smtp;
      smtp_auth plain;
    }`

Updated version that appears to work for Outlook for Mac:
`    server {
      listen 465 ssl;
      listen [::]:465 ssl;
      protocol smtp;
      smtp_auth plain login;
    }`

**Comment by muhlemmer** [comment]
Hey guys, thanks for the suggestions. If those changes work for you, feel free to submit a PR. 
I've [updated the documentation](https://github.com/Mailu/Mailu/blob/master/docs/contributors/environment.rst) to include the workflow on git. It is included in master, but not yet published on mailu.io.

After submitting a PR, your suggestions will be tested an reviewed by the team. If it causes no issues, it will be merged.

Personally, I think @developer011  is a cleaner approach, but I'm not sure if this would work for windows live mail as well.

**Comment by muhlemmer** [comment]
Windows live mail is [discontinued and unsupported](https://support.microsoft.com/en-us/help/18614/windows-essentials) by Microsoft since January 2017. I found this out when I tried to install it on my windows-7 vm.

> Windows Essentials 2012 suite reached end of support on January 10, 2017.
> ....
> Windows Essentials 2012 included ...., Windows Live Mail, ....
 
As mentioned in @chrisch-hh 's post:
> Other (more modern) mail clients that I've tested work without any problems (e.g. Thunderbird 60, Outlook 2016, Android Mail).

So I don't see the necessity to implement his suggestions.

However, @developer011 seems to have the [brand-new outlook 2019](https://social.technet.microsoft.com/wiki/contents/articles/31133.outlook-and-outlook-for-mac-update-file-versions.aspx#G). So as such, I will implement his suggested change. However, I don't have a mac, windows-10 or any office license to test this myself.
